src/EnergyPlus/HeatPumpWaterToWaterSimple.cc

```diff
@@ -770,6 +1359,7 @@ namespace HeatPumpWaterToWaterSimple {
 		func4 = ( SourceSideMassFlowRate / ( SourceSideVolFlowRateRated * rhoSourceSide ) );
 
 		QLoad = CoolCapRated * ( CoolCapCoeff1 + ( func1 * CoolCapCoeff2 ) + ( func2 * CoolCapCoeff3 ) + ( func3 * CoolCapCoeff4 ) + ( func4 * CoolCapCoeff5 ) );
+// this doesn't match eng ref, doc is missing CoolPowerRated
```

Comment:
Is this comment addressed above in the docs then?  If so, the comment should go away.  If it still needs resolution, OK.